tests/GeneratorTest.php

```diff
@@ -1,25 +1,29 @@
 <?php
 
+declare(strict_types=1);
+
 use PHPUnit\Framework\TestCase;
 use Fontis\BpayRefGenerator\Generator;
```

Comment:
The list of `use` statements should be sorted alphabetically. Apologies for not brining this up in my previous set of comments.